main.py

```diff
@@ -16,64 +16,68 @@
         {"name": "viewport", "content": "width=device-width, initial-scale=1.0"}
     ],
 )
-initial_days = 7
-
-import constants
 
-df =  constants.NYC_BIKE_API_LINK
+days = 30
```

Comment:
Should days be in the constants file? If we are resurrecting the dropdown later, it could make sense to leave here.